source/fundamentals/linq.txt

```diff
@@ -0,0 +1,566 @@
+.. _csharp-linq:
+
+====
+LINQ
+====
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+In this guide you can learn how to use
+`LINQ <https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/?redirectedfrom=MSDN>`__
+with the {+driver-long+}. LINQ allows you to construct queries against
+strongly typed collections of objects by using language keywords and operators.
+The {+driver-short+} automatically translates LINQ queries into
+:manual:`aggregation operations </aggregation>`.
+
+The examples in this guide use the ``restaurants`` collection
+in the ``sample_restaurants`` database provided in the :atlas:`Atlas sample datasets </sample-data>`.
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets,
+see the :ref:`<csharp-quickstart>`.
+
+The following ``Restaurant`` class models the documents in this collection:
+
+.. literalinclude:: /includes/fundamentals/code-examples/linq.cs
+   :language: csharp
+   :dedent:
+   :start-after: start-restaurant-model
+   :end-before: end-restaurant-model
+
+.. note::
+
+   The documents in the ``restaurants`` collection use the camel
+   case naming convention. The examples in this guide use a ``ConventionPack``
+   to deserialize the fields in the collection into Pascal case and map them to
+   the properties in the ``Restaurant`` class.
+
+   To learn more about custom serialization see :ref:`csharp-custom-serialization`.
+
+.. _csharp-linq-queryable:
+
+Make A Collection Queryable
+---------------------------
+
+To use LINQ to query your collection, you must first convert the collection to
+an `IQueryable
+<https://learn.microsoft.com/en-us/dotnet/api/system.linq.iqueryable?view=net-7.0>`__
+instance. To convert your collection, use the ``AsQueryable`` method
```

Comment:
How about instead of "to convert your collection..."

   to get an IMongoQueryable object pointing at your collection

Feel free to argue back. Maybe "looser" language is acceptable to readers.

Reply:
I changed the wording here, let me know if this feels better to you